aspnetcore/mvc/controllers/dependency-injection/sample/ControllerDI/Views/Shared/_CookieConsentPartial.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using Microsoft.AspNetCore.Http.Features
+
```

Comment:
This might be in the template, but no blank line here if going for total consistency.